anycloud/cli/src/deploy/mod.rs

```diff
@@ -524,10 +528,43 @@ pub async fn add_deploy_config() {
       .with_prompt("Virtual machine type")
       .interact_text()
       .unwrap();
+    let min_replicas: Option<u32>;
```

Comment:
in Rust it's preferred to instantiate variables wherever they're declared. So this line could be put after the next statement as:
```rust
let min_replicas: Option<u32> = if let Ok(replicas) = replicas.parse::<u32>() {
  Some(replicas)
} else { ... };
```

Reply:
Another option would be something like this:
```rust
let min_replicas = replicas.parse::<u32>().unwrap_or_else(|_| {
  eprintln!(...);
  exit(1);
});
```

Reply:
Got it

Reply:
Why not use the option map here? If it is `Some` it will transform the data and if it is `None` it skips your callback entirely.

Reply:
it could be something like this?
```rust
let min_replicas: Option<u32> = replicas.parse::<u32>().into::<Option<_>>().unwrap_or_else(...);
```

Reply:
Applied second suggestion